the-blue-alliance/the-blue-alliance-android
# Issue 780

**Summary:** 
`ViewModelRenderer. renderToViewModel()` is declared `@Nullable`, so don't crash if it returns null.

**Issues Reference:** 
#780 

**Test Plan:** 
I tested this on a Nexus 5X N emulator. Steps:

1. Temporarily modify `DistrictPointsUpdatedNotification.renderToViewModel()` to return `null` like it used to.
2. Run the app.
3. Navigate to any screen except Recent Notifications.
4. From the adb command line, run `scripts/test_notification.py district-points-updated`. --> The app should display a notification.
5. In the app, navigate to the Recent Notifications screen. --> With this fix, the app should log a warning for each `null` DistrictPointsUpdatedNotification ViewModel stored in its recent list, instead of crashing as in #780.
